src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[fromFormat.length - 1]);
+
+  const result = [];
+
+  for (let i = 0; i < toFormat.slice(0, 3).length; i++) {
+    const value = toFormat[i];
+    let currentValue = value;
+
+    if (value === 'YYYY') {
+      currentValue = 'YY';
```

Comment:
Consider will be better if your create a variables for these magic words and use them everywhere